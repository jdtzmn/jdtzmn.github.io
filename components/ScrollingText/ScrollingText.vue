<template lang="pug">
  .scrolling-text
</template>

<script lang="ts">
import Vue from 'vue'
export default Vue.component('scrolling-text', {})
</script>

<style lang="scss" scoped>
@import '~@/styles/colors';

.scrolling-text {
  background: $lightgray;
  height: 300px;
}
</style>
