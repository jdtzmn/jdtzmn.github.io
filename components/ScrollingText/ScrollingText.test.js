import { shallowMount } from '@vue/test-utils'
import ScrollingText from './ScrollingText'

describe('ScrollingText', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallowMount(ScrollingText)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should render scrolling-text div', () => {
    expect(wrapper.contains('.scrolling-text')).toBeTruthy()
  })
})
