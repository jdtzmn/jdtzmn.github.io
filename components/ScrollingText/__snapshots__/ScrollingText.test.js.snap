// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollingText should match snapshot 1`] = `<div class="scrolling-text"></div>`;
